examples/spec/integration/child_id_reuse_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'workflows/parent_id_reuse_workflow'
```

Comment:
nit: can you please match the name of the file to the name of the workflow that is being tested?